### PR TITLE
Added parser for abrt CCpp.conf

### PR DESCRIPTION
**Pull request by vishwanathjadhav**
Added  AbrtCCppConf  parser class for  /etc/abrt/plugins/CCpp.conf

### CODE REVIEW

**Comment by vishwanathjadhav** [comment]
@xiangce - Please review the parsers

**Comment by xiangce** [comment]
@vishwanathjadhav - The code looks good, but please fix the flake8 errors at first.

**Review by xiangce — APPROVED** [review]
approved pending the approval of spec

**Pull request merged**
